### DISCUSSION TITLE
Default CSS Utility Not Taking Affect

### DISCUSSION THREAD

**Discussion by donhook** [tailwindlabs/tailwindcss#5425] [Help] [answered]
I am on the latest version of TailwindCss and am not sure what I am doing wrong. 

I have a background color that should take place by default, no matter what the view size is.  However, it is only taking affect once the view size exceeds lg.  

Here is the div.  The utility class is bg-secondary, which should display the defined background color on all sizes, but is only displaying on LG.

        <div className="flex absolute bg-secondary opacity-70 lg:top-2/3 lg:p-24 lg:left-30">
          &nbsp;
        </div>

Thanks in advance.

<div className="flex absolute bg-secondary opacity-70 lg:top-2/3 lg:p-24 lg:left-30">
  ....
</div>

**Comment by Rodriguezvd** [comment]
Someone else just posted a related question, and I've been wondering this myself. Being new to serious front end dev (been doing mediocre PHP for over a decade), I was made aware of CSS resets, but I've not been able to determine what CSS reset is the best. Yes, I could try them all out, but I would rather just ask Reddit :)

Is Normalize.css the go-to solution?


[rapidfs official](https://www.rapidfs.us/)

**Comment by simonswiss** [comment, marked as answer]
Hey @donhook 👋

Not sure of the context of your div in this case, but the background of the element _is_ actually showing.

Because the content of the div is just `&nbsp;`, and the position is `absolute`, the background will only be a tiny width and at the top-left of the relative container, up to the `lg` breakpoint:

<img width="1202" alt="Pasted_Image_16_9_21__1_23_pm" src="https://user-images.githubusercontent.com/485747/133544367-3c8d54de-b087-46cf-97f6-476d5893ee90.png">

The shape becomes much bigger from the `lg` breakpoint, when you inform it to have some padding. It's also positioned 2/3 from the top and `-30` from the left, as expected:

![image](https://user-images.githubusercontent.com/485747/133544517-3e196855-a67f-4ec1-934c-feaed27c5cbe.png)

https://play.tailwindcss.com/Z0Wzagv3O8

**Reply by donhook** [reply]
Thank you @simonswiss
